### PR TITLE
Cloud changelog Jan 12

### PR DESCRIPTION
**Pull request by amyblais**
Changelog for Jan 12th cloud release.

### CODE REVIEW

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 1]
```diff
@@ -2,6 +2,43 @@
...
+ - Added ``webhook create`` endpoints to local mode and the ability to create webhooks for other users.
+ - Added a mattermost command to initialize the database.
+ - Enabled ``ExperimentalDataPrefetch`` for all servers and removed the corresponding setting.
+ - Added support for processing import files through the API.
+ - Added support for protocol relative URLs while using the Image Proxy.
+ - Added shared channels and ``remote_cluster_service`` under a license check.
+ - A Striped LRU cache is now used by default.
+
+### Bug Fixes
+ - Fixed an issue where the permissions of a System Role got deleted when changing the access level to any permission.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:28]:
```suggestion
 - Fixed an issue where the permissions of a System Admin Role got deleted when changing the access level to any permission.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 2]
```diff
@@ -2,6 +2,43 @@
...
+ - Fixed an issue where editing a ``/me`` post behaved differently in Webapp and Mobile App.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:29]:
```suggestion
 - Fixed an issue where editing a ``/me`` post behaved differently within the Mattermost Web App and the Mobile App.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 3]
```diff
@@ -2,6 +2,43 @@
...
+ - Fixed an issue where the hover state did not span the whole width of the left-hand side on category headers.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:30]:
```suggestion
 - Fixed an issue where the hover state on category headers did not span the whole width of the left-hand navigation.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 4]
```diff
@@ -2,6 +2,43 @@
...
+ - Fixed an issue where plugins on the left-hand side of the System Console were sorted differently than the ones in the Plugin Management page.
+ - Fixed an issue where 15-character team names were truncated when the experimental sidebar was enabled.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:32]:
```suggestion
 - Fixed an issue where 15-character team names were truncated when the experimental channel sidebar was enabled.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 5]
```diff
@@ -2,6 +2,43 @@
...
+ - Fixed an issue where the sidebar menus weren't styled correctly on mobile web view.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:33]:
```suggestion
 - Fixed an issue where the sidebar menus weren't styled correctly in the Mattermost Mobile App.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 6]
```diff
@@ -2,6 +2,43 @@
...
+ - Fixed an issue where jumping into an archive channel and clicking to jump to recent messages sent the user out of the archived channel.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:34]:
```suggestion
 - Fixed an issue where jumping into an archive channel and clicking the link to jump to recent messages sent the user out of the archived channel.
```

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 7]
```diff
@@ -2,6 +2,43 @@
...
+ - Fixed an issue where the tooltip text for copying an incoming webhook url was unclear.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:35]:
```suggestion
 - Fixed an issue where the tooltip text for copying an incoming webhook URL was unclear.
```

**Review by cwarnermm — COMMENTED** [review]
Comments/suggestions provided inline.

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 8]
```diff
@@ -2,6 +2,43 @@
...
+## Release 2021-01-12
+
+### Highlights
+ - Pre-packaged and pre-installed the Incident Management v1.2.0.
+
+### Improvements
+
+#### User Interface (UI)
+ - Changed number of files allowed per post, from 5 to 10.
+ - Enabled the 3-dot menu on channels in the new sidebar to move multi-selected groups of channels.
```
[the remark this answers is not in the document]
Comment (wiersgallak) [reply source/cloud/cloud-administration/cloud-changelog.md:14]: @andrewbrown00 may have some input on this as well.

[the remark this answers is not in the document]
Comment (andrewbrown00) [reply source/cloud/cloud-administration/cloud-changelog.md:14]: Hi @cwarnermm, we can refer to it as the **More options** icon. We can also consider introducing images in the documentation to have a supporting visual.

**Example**
Enabled the **More options** image menu on channels...

**Review by cwarnermm — APPROVED** [review]
Well done! Looks great!

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 9]
```diff
@@ -2,6 +2,42 @@
...
+## Release 2021-01-12
+
+### Highlights
+ - Pre-packaged and pre-installed Mattermost Incident Management v1.2.0.
+
+### Improvements
+
+#### User Interface (UI)
+ - Changed the number of file attachments allowed per post, from 5 to 10.
+ - Enabled the More options menu on channels in the new sidebar to move multi-selected groups of channels.
```
Comment (cwarnermm) [inline source/cloud/cloud-administration/cloud-changelog.md:14]:
```suggestion
 - Enabled the **More options** menu on channels in the new sidebar to move multi-selected groups of channels.
```

**Review by wiersgallak — APPROVED** [review]

**Review by andrewbrown00 — COMMENTED** [review]
@cwarnermm should we break this in to separate items or call out the **multi-select** more?

I don't see any reference in the current changelog that we added **multi-select** support in the LHS. Wondering if we should be explicit about the functionality?

File: source/cloud/cloud-administration/cloud-changelog.md [hunk 10]
```diff
@@ -2,6 +2,42 @@
...
+## Release 2021-01-12
+
+### Highlights
+ - Pre-packaged and pre-installed Mattermost Incident Management v1.2.0.
+
+### Improvements
+
+#### User Interface (UI)
+ - Changed the number of file attachments allowed per post, from 5 to 10.
+ - Enabled the **More options** menu on channels in the new sidebar to move multi-selected groups of channels.
```
Comment (amyblais) [inline source/cloud/cloud-administration/cloud-changelog.md:14]: @andrewbrown00 Would something like this work better:

"Added support for multi-selecting channels via the **More options** menu in the channel sidebar to be able to move multi-selected groups of channels to a category."

Comment (cwarnermm) [reply source/cloud/cloud-administration/cloud-changelog.md:14]: @amyblais - Was multi-select channel support in LHS added to a previous Cloud release changelog? I'm documenting this functionality for release v5.31, which suggests to me that it's already released for Cloud.

@amyblais - I'd avoid a statement that implies that selecting multiple channels at a time is something that happens through the **More options** menu. Instead, I'd recommend something along the lines of:

"Added support to move multi-selected groups of channels to another category via the **More options** menu.

Thoughts?

Comment (amyblais) [reply source/cloud/cloud-administration/cloud-changelog.md:14]: Yes, I see, multi-selection in the changelog for the previous versioning: https://docs.mattermost.com/cloud/cloud-administration/cloud-changelog.html.

In that case your suggestion for wording makes sense, thanks!

Comment (andrewbrown00) [reply source/cloud/cloud-administration/cloud-changelog.md]: Thanks for the quick feedback and suggestions here 🙂 

@amyblais let's go with @cwarnermm's suggestion:

> Added support to move multi-selected groups of channels to another category via the More options menu.

**Pull request merged**